x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/ssl/SSLReloadDuringStartupIntegTests.java

```diff
@@ -23,22 +25,37 @@
 import java.nio.file.StandardCopyOption;
 import java.util.concurrent.CountDownLatch;
 
+/**
+ * Start a 3 node cluster, with one node having a bad Transport keystore setting.
```

Comment:
I don't think this test always uses 3 nodes, it should be random unless explicitly defined via an annotation. 

https://github.com/elastic/elasticsearch/blob/main/test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java#L1730

If the test requires 3 for correctness, maybe that is the issue with this test ?

Reply:
I changed the comment. The number of nodes is randomized by the super class.

Reply:
Done